ceph_salt/config_shell.py

```diff
@@ -338,32 +359,22 @@ def possible_values(self):
             }
         }
     },
-    'Deployment': {
+    'Cluster_Bootstrap': {
```

Comment:
maybe `Cephadm_Bootstrap` would be more explicit and easier for the user to understand?

and `Ceph_Cluster` instead of `Cluster`

Just an idea.